be/src/vec/exec/scan/file_scanner.cpp

```diff
@@ -1480,6 +1489,21 @@ Status FileScanner::_generate_parititon_columns() {
     return Status::OK();
 }
 
+Status FileScanner::_generate_data_lake_partition_columns() {
+    _data_lake_partition_col_descs.clear();
+    if (_current_range.__isset.data_lake_partition_values) {
```

Comment:
data_lake_partition_values maybe null?
Because on FE side, if we failed to deserialize the partition value, a null will be returned and set to `data_lake_partition_values`